code/data_cleaning.R

```diff
@@ -205,12 +205,53 @@ data <- data %>%
   ) %>%
   mutate(standardized_facility = ifelse(facility_name_title_case %in% c(
     "Ssam / Dhs", "Centru Saude Deho", "Ps Estado", "Postu Da Saude",
-    "Chc Centru Saude", "Js200921", "Hp", "Hp Deleco", "Hp Ekapu", "Ps Fatuquero",
```

Comment:
Yes, I see that `Ps Fatuquero` is actually a facility! But this also needs to be added to the manual mapping above in line 197.